### PR TITLE
ceph-daemon -> cephadm, mgr/ssh -> mgr/cephadm

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

File: doc/mgr/cephadm.rst [hunk 1]
```diff
@@ -1,13 +1,13 @@
-================
-SSH orchestrator
-================
+====================
+cephadm orchestrator
+====================
 
-The SSH orchestrator is an orchestrator module that does not rely on a separate
+The cephadm orchestrator is an orchestrator module that does not rely on a separate
```
Comment (sebastian-philipp) [inline doc/mgr/cephadm.rst]: Eventually we want to mention the CLI tool here as well.

Comment (liewegas) [reply doc/mgr/cephadm.rst]: this is changed in a later commit in the series

**Comment by liewegas** [comment]
Make check passes and it appears to be okay locally.  We're basically blocked by https://github.com/ceph/ceph-container/pull/1533 or something similar.  We can either merge this now and the tests will be broken until ceph-container is updated (~+ a day to get latest-master-devel to update), or wait for the ceph-container change to merge (breaking master containers) and merge this after.  The net is basically the same I think, so I'm inclined to just merge this now!

**Comment by epuertat** [comment]
@liewegas Any hint on how to pronounce it? ceph-A-D-M, ceph-adm(in), ...?

**Comment by liewegas** [comment]
I like 'ceph ay dee em' but 'ceph admin' works too?

**Pull request merged**